packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -118,7 +118,8 @@ module.exports = {
           // Point ESLint to our predefined config.
           options: {
             configFile: path.join(__dirname, '../.eslintrc'),
-            useEslintrc: false
+            useEslintrc: false,
+            cache: true
```

Comment:
We probably don't want this specific setting to be inside of `// @remove-on-eject` block.